File: agent/python/perper/protocol/standard/perper_agent.py
from collections import OrderedDict
from pyignite import GenericObjectMeta
from pyignite.datatypes import String


class PerperAgent(
    metaclass=GenericObjectMeta,
    type_name="PerperAgent",
    schema=OrderedDict(
        [
            ("Agent", String),
            ("Instance", String)
        ]
    ),
):
    pass

File: agent/dotnet/src/Perper/Protocol/Cache/Standard/PerperAgent.cs
namespace Perper.Protocol.Cache.Standard
{
    public class PerperAgent
    {
        public PerperAgent(
            string agent,
            string instance)
        {
            Agent = agent;
            Instance = instance;
        }

        public string Agent { get; }
        public string Instance { get; }
    }
}